scripts/main/build.js

```diff
@@ -264,7 +264,7 @@ build.imageview = function (data, visibleControls) {
 			let medium = '';
 
 			if (data.medium2x && data.medium2x !== '') {
-				medium = `srcset='${data.medium} ${parseInt(data.medium_dim, 10)}w, ${data.medium2x} ${parseInt(data.medium2x_dim, 10)}w'`;
+				medium = `class="lazyload" src="dist/images/placeholder.png" data-srcset='${data.medium} ${parseInt(data.medium_dim, 10)}w, ${data.medium2x} ${parseInt(data.medium2x_dim, 10)}w'`;
```

Comment:
You don't need the class lazyload here because you are in the image view. As such there is only one picture to load.

Reply:
Got it - I completely removed the lazy loading here since there is no point if it's only 1 image